api4/system.go

```diff
@@ -183,6 +183,17 @@ func getSystemPing(c *Context, w http.ResponseWriter, r *http.Request) {
 		w.Header().Set(filestoreStatusKey, s[filestoreStatusKey])
 	}
 
+	if deviceID := r.FormValue("device_id"); deviceID != "" {
+		canSend, err := c.App.TestPushNotification(deviceID)
```

Comment:
Forgot to change it here?

Reply:
Trusted too much in the "rename symbol".